<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid rgb(239, 88, 86);
    border-radius: 15px;
    width: 26px;
    height: 26px;
    font-size: 16px;
    font-weight: bold;
    color: rgb(239, 88, 86);
    qproperty-text: &quot;х&quot;;
}

QPushButton:hover {
    border-color: #495057;
    color: #495057;
    background-color: #f8f9fa;
}

QPushButton:pressed {
    background-color: #e9ecef;
    border-color: #343a40;
    color: #343a40;
}</string>
       </property>
       <property name="text">
        <string>←</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chck_music">
     <property name="styleSheet">
      <string notr="true">QCheckBox {
        color: #2d3436;
        font-size: 14px;
        padding: 12px 16px;
        spacing: 16px;
        border-radius: 8px;
    }
QCheckBox:hover {
    background-color: #f8f9fa;}</string>
     </property>
     <property name="text">
      <string>Музыка</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chck_sounds">
     <property name="styleSheet">
      <string notr="true">QCheckBox {
        color: #2d3436;
        font-size: 14px;
        padding: 12px 16px;
        spacing: 16px;
        border-radius: 8px;
    }
QCheckBox:hover {
    background-color: #f8f9fa;}</string>
     </property>
     <property name="text">
      <string>Звуковые эффекты</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid #868e96;
    border-radius: 15px;
    width: 30px;
    height: 30px;
    font-size: 16px;
    font-weight: bold;
    color: #868e96;
    qproperty-text: &quot;i&quot;;
}

QPushButton:hover {
    border-color: #495057;
    color: #495057;
    background-color: #f8f9fa;
}

QPushButton:pressed {
    background-color: #e9ecef;
    border-color: #343a40;
    color: #343a40;
}</string>
       </property>
       <property name="text">
        <string>i</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
